trace_decoder/src/decoding.rs

```diff
@@ -545,8 +546,14 @@ fn process_txn_info(
         delta_out,
     )?;
 
+    let burn_addr = match cfg!(feature = "cdk_erigon") {
```

Comment:
```suggestion
    let burn_addr = match use_burn_addr {
```
Let's move this to a runtime flag - `trace_decoder` no longer needs to have a feature :)

Reply:
I'm sorry, I am a bit confused: if I remove the feature from trace_decoder side, how do view the link between trace_decoder and zk_evm? What I mean is: if we enable `--use-burn-addr` on the trace_decoder side, but but do not use `--features cdk_erigon`, then the feature is not activated, and so it is ignored on the zk_evm side, right? So I was wondering whether you had an idea on how to avoid this?